tests/system_tests/psyclone/test_psyclone_system_test.py

```diff
@@ -160,17 +161,17 @@ def test_run(self, config):
             config.build_output / 'algorithm/algorithm_mod_psy.f90',
 
             # Expect these prebuild files
-            # todo: the kernal hash differs between fpp and cpp, perhaps just use wildcards.
-            config.prebuild_folder / 'algorithm_mod.1602753696.an',  # x90 analysis result
-            config.prebuild_folder / 'my_kernel_mod.4187107526.an',  # kernel analysis results
-            config.prebuild_folder / 'algorithm_mod.5088673431.f90',  # prebuild
-            config.prebuild_folder / 'algorithm_mod_psy.5088673431.f90',  # prebuild
+            # The kernel hash differs between fpp and cpp, so just use wildcards.
+            config.prebuild_folder / 'algorithm_mod.*.an',  # x90 analysis result
+            config.prebuild_folder / 'my_kernel_mod.*.an',  # kernel analysis results
+            config.prebuild_folder / 'algorithm_mod.*.f90',  # prebuild
+            config.prebuild_folder / 'algorithm_mod_psy.*.f90',  # prebuild
         ]
 
-        assert all(not f.exists() for f in expect_files)
+        assert all((glob.glob(str(f)) == []) for f in expect_files)
```

Comment:
There is a "glob" method on `Path` objects. Might be neater than casting to string?

Reply:
I actually discussed this with Jason, the main issue we had was that while `Path` has a `glob` method, it doesn't work the way I expected it to do: 
```
p = Path("/tmp/*.txt")
p.glob()   # error :(
```
because `glob` requires a pattern, it does not automatically take the pattern from the path.

So assuming that you are ok with it I will change this as:
```
p.parent.glob(p.name) ...
```
This seems to work

Reply:
I had to use a `list(...)`, since a generator (which is returned from Path's glob method) cannot easily be tested to see if it is empty.